pages/productAppExtension/[productId]/index.tsx

```diff
@@ -0,0 +1,39 @@
+import { H4, Panel, Text } from "@bigcommerce/big-design";
+import { useRouter } from "next/router";
+import ErrorMessage from "@components/error";
+import Loading from "@components/loading";
+import { useProductInfo } from "@lib/hooks";
+
+const ProductAppExtension = () => {
+    const router = useRouter();
+    const productId = Number(router.query?.productId);
+    const list = [];
+    const { error, isLoading, product } = useProductInfo(productId, list);
```

Comment:
since we've eliminated the previous hook, there is no longer a `list` to pass in here

```suggestion
    const { error, isLoading, product } = useProductInfo(productId);
```